Cannot find module '@babel/preset-typescript'

## 🐛 Bug Report

Upgrading from alpha 43 to 44 in an existing project yields an error detailing a missing module, namely `@babel/preset-typescript`.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)

Yes

## To Reproduce

1. Be on an existing alpha 43 site
1. Upgrade the classic preset and core modules
1. Run `yarn start`

## Expected behavior

<!--
  How did you expect your project to behave?
  It’s fine if you’re not sure your understanding is correct.
  Write down what you thought would happen.
-->

I expect the app to compile.

## Actual Behavior

<!--
  Did something go wrong?
  Is something broken, or not behaving as you expected?
  Describe this section in detail, and attach screenshots if possible.
  Don't only say "it doesn't work"!
-->

The app fails to compile and results in an error:

```
phoqe@ubiq docs % yarn start
yarn run v1.22.0
$ docusaurus start
Starting the development server...

✖ Client
  Compiled with some errors in 1.36s

ℹ ｢wds｣: Project is running at http://localhost:3000/
ℹ ｢wds｣: webpack output is served from /
ℹ ｢wds｣: Content not from webpack is served from /Users/phoqe/Projects/rmuif/docs
ℹ ｢wds｣: 404s will fallback to /index.html


./node_modules/@docusaurus/core/lib/client/clientEntry.js
Error: Cannot find module '@babel/preset-typescript' from '/Users/phoqe/Projects/rmuif/docs'
    at Array.map (<anonymous>)
    at cachedFunction.next (<anonymous>)
 @ multi ./node_modules/react-dev-utils/webpackHotDevClient.js ./node_modules/@docusaurus/core/lib/client/clientEntry.js main[1]
```

![image](https://user-images.githubusercontent.com/7033377/76169416-27355d80-6178-11ea-8874-9cdbf5031882.png)

## Your Environment

<!-- Include as many relevant details about the environment you experienced the bug in -->

- Docusaurus version used: 2.0.0-alpha.44
- Environment name and version (e.g. Chrome 78.0.3904.108, Node.js 10.17.0): Brave Version 1.4.96 Chromium: 80.0.3987.132 (Official Build) (64-bit), v13.8.0
- Operating system and version (desktop or mobile): macOS Catalina 10.15.3 (19D76)

## Reproducible Demo

https://github.com/rmuif/docs

1. Clone repo
2. Install dependencies
3. `yarn start`

<!--
  What happens if you skip this step?

  Someone will read your bug report, and maybe will be able to help you,
  but it’s unlikely that it will get much attention from the team. Eventually,
  the issue will likely get closed in favor of issues that have reproducible demos.

  Please remember that:

    * Issues without reproducible demos have a very low priority.
    * The person fixing the bug would have to do that anyway. Please be respectful of their time.
    * You might figure out the issues yourself as you work on extracting it.

  Thanks for helping us help you!
-->

--- a/packages/docusaurus/package.json
+++ b/packages/docusaurus/package.json
@@ -37,6 +37,7 @@
     "@babel/plugin-transform-runtime": "^7.7.4",
     "@babel/preset-env": "^7.7.4",
     "@babel/preset-react": "^7.7.4",
+    "@babel/preset-typescript": "^7.7.4",
     "@babel/runtime": "^7.7.4",
     "@docusaurus/utils": "^2.0.0-alpha.44",
     "@endiliey/static-site-generator-webpack-plugin": "^4.0.0",

